de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingReferencePointNegotiation.java

```diff
@@ -31,19 +32,19 @@
 import de.fu_berlin.inf.dpp.util.CoreUtils;
 
 /**
- * Implementation of {@link AbstractIncomingProjectNegotiation} utilizing a
- * transferred zip archive to exchange differences in the project files.
+ * Implementation of {@link AbstractIncomingReferencePointNegotiation} utilizing
+ * a transferred zip archive to exchange differences in the project files.
  */
-public class ArchiveIncomingProjectNegotiation extends
-    AbstractIncomingProjectNegotiation {
+public class ArchiveIncomingReferencePointNegotiation extends
+    AbstractIncomingReferencePointNegotiation {
 
     private static final Logger LOG = Logger
-        .getLogger(ArchiveIncomingProjectNegotiation.class);
+        .getLogger(ArchiveIncomingReferencePointNegotiation.class);
 
-    public ArchiveIncomingProjectNegotiation(
+    public ArchiveIncomingReferencePointNegotiation(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 11 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMeiDZywvPe4LUfS&open=AWZfKMeiDZywvPe4LUfS)

[ISSUE_KEY]: AWZfKMeiDZywvPe4LUfS